src/app/core/entity-details/form/form.component.ts

```diff
@@ -82,6 +105,25 @@ export class FormComponent<E extends Entity> implements FormConfig, OnInit {
     this.entityFormService.resetForm(this.form, this.entity);
     this.form.disable();
   }
+
+  private filterFieldGroupsByPermissions() {
```

Comment:
Do you see any way on how we can move (most of) this logic into the `EntityFormService` or the `EntityFormComponent` so we can re-use the functionality in all forms across the map?

Reply:
Logic moved to EntityFormComponent and EntityFormService